terraform/cloud-platform-components/prometheus.tf

```diff
@@ -61,6 +61,10 @@ resource "helm_release" "prometheus_operator" {
     "null_resource.deploy",
   ]
 
+  provisioner "local-exec" {
+    command = "kubectl apply -n monitoring -f ${path.module}/resources/prometheusrule-alerts/"
```

Comment:
You'll need to define a local-exec with the delete command.

Reply:
I did not add it, as part of the uninstall of operator all rules get deleted including all the custom rules, so when the delete command runs, it does not find anything to delete

Reply:
I think we should add it then with the `--ignore-not-found` flag. So when we fix that CRD issue (which might have been addressed upstream) it just works properly.

Additionally, should this be a separate `null_resource` instead? I think it would improve lifecycle management (you can `taint / apply / destroy` independently from the prom chart). Is there a reason why this should be different to the proxy, for example?